From 2feb2d987dbbc690fd3194d129cc0811d111d6f6 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Fri, 19 Oct 2018 12:10:58 +0200
Subject: [PATCH] Merge android-9.0.0_r12

commit f16d5f8b210c9f12260594099515646a5574439a
Merge: 492fc0c 19dfcac
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Sun Jul 8 03:06:39 2018 +0000

    Snap for 4880674 from 19dfcac7421c0c91433a48e20eebf681e2c255f8 to pi-dr1-release

    Change-Id: Ie6f7a9c48428423215011b4bc27d927139bfb51a

commit 19dfcac7421c0c91433a48e20eebf681e2c255f8
Author: Bill Yi <byi@google.com>
Date:   Thu Jul 5 00:55:02 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: I31180ea0be44bf409b1a6e248debec4cf553fcf8

commit 492fc0c92c4604e888ba72b2c32f5dd8e6444d74
Merge: b67aa36 dba520d
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Tue Jun 5 03:02:03 2018 +0000

    Snap for 4820872 from dba520d6c6f936d6fd82c60d28877ed0528f96db to pi-dr1-release

    Change-Id: I3775c6447ed89a0fd1ee366d0c0e718329e9a3a6

Change-Id: I3f4584f5d7b594bc67956ab99e37149b4bdd9a7f
---
 res/values-sw/strings.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/values-sw/strings.xml b/res/values-sw/strings.xml
index 8fac74b..765a19d 100644
--- a/res/values-sw/strings.xml
+++ b/res/values-sw/strings.xml
@@ -22,7 +22,7 @@
     <string name="description_delete_button" msgid="88733669653753441">"futa"</string>
     <string name="description_image_button_plus" msgid="7821061347207448362">"jumlisha"</string>
     <string name="description_voicemail_button" msgid="6281470684653439120">"ujumbe wa sauti"</string>
-    <string name="default_notification_description" msgid="6011631627871110785">"Sauti chaguo msingi (<xliff:g id="DEFAULT_SOUND_TITLE">%1$s</xliff:g>)"</string>
+    <string name="default_notification_description" msgid="6011631627871110785">"Sauti chaguomsingi (<xliff:g id="DEFAULT_SOUND_TITLE">%1$s</xliff:g>)"</string>
     <string name="callFailed_userBusy" msgid="5163219086553610486">"Simu inatumika"</string>
     <string name="callFailed_congestion" msgid="3645995103907418337">"Mtandao unatumika"</string>
     <string name="callFailed_timedOut" msgid="7772951976278867294">"Hakuna jibu, muda umekwisha"</string>
-- 
2.17.1

